<launch>

    <!-- Turtlesim Node-->
    <node name="sim" pkg="turtlesim" type="turtlesim_node"/>

    <!-- Turtlesim Teleoperation-->
    <node name="teleop" pkg="turtlesim" type="turtle_teleop_key"  output="screen" launch-prefix="xterm -e"/>

    <!-- Turtlesim Start-->
    <node name="start" pkg="practica_acciona" type="start_snake_service.py"  output="screen"/>

    <!-- Turtlesim Start-->
    <node name="broadcaster" pkg="practica_acciona" type="turtle_broadcaster2.py"  output="screen"/>

    <!-- Turtlesim Start-->
    <node name="controller" pkg="practica_acciona" type="turtle_controller2.py"  output="screen"/>

</launch>